package ar.edu.utn.frlp.proyecto.grupo10.mi_utn.repository;

import ar.edu.utn.frlp.proyecto.grupo10.mi_utn.model.Schedule;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.Query;
import org.springframework.stereotype.Repository;
import org.springframework.web.bind.annotation.ResponseBody;

import java.util.List;

@Repository
public interface SchedulesRepository extends JpaRepository<Schedule, Long> {

    @Query("SELECT DISTINCT s.classroom FROM Schedule s")
    List<String> findAllDistinctClassrooms();
}
